src/reynir_correct/pattern.py

```diff
@@ -1707,6 +1707,90 @@ def né(self, match: SimpleTree) -> None:
             )
         )
 
+    def agreement_conj(self, match: SimpleTree) -> None:
+        vp = match.first_match("VP > (so_ft|so_et)")
```

Comment:
Would be nice to have a comment here that describes exactly what this rule is correcting, preferably with an example.